lib/stream-concat.js

```diff
@@ -0,0 +1,56 @@
+const Transform = require('stream').Transform;

```

Comment:
Is there no stream module on NPM which does this already? If not, I think we should publish this as an individual module as it has lots of uses outside of this codebase.